src/app.ts

```diff
@@ -277,3 +278,5 @@ if (ENV.DEBUG_THEME) {
 
 initDeviceSettings().catch(err => console.log(err))
 initInfoServer().catch(err => console.log(err))
+
+global.Buffer = global.Buffer || Buffer
```

Comment:
This approach seems dangerous, so maybe don't touch the globals unless we need to:

```ts
if (global.Buffer == null) global.Buffer = Buffer
```

Reply:
changed